themes/bootstrap5/js/common.js

```diff
@@ -469,6 +488,11 @@ var VuFind = (function VuFind() {
     _theme = theme;
   };
 
+  /**
+   * Sets up click handlers for QR code links to display the QR code image on click.
+   * @param {Element} [_container] The container element to search for links; defaults to (default = document.body).
```

Comment:
I think some extra words can be removed here:

```suggestion
   * @param {Element} [_container] The container element to search for links (default = document.body).
```